reference/constraints/Range.rst

```diff
@@ -320,15 +320,15 @@ Options
 min
 ~~~
 
-**type**: ``int`` or ``string`` (date format)
+**type**: ``numeric`` or ``string`` (date format)
```

Comment:
we should call it `number` IMHO.

Reply:
I was doubtful indeed. Is true that `number` is better but `numeric` gives the reader back additional infos somehow. Don't know if `number` (same for `numeric` of course) is used elsewhere in docs. Whe should try adopt that standard (if any)